.github/workflows/docker.yml

```diff
@@ -1,8 +1,10 @@
 name: Docker
 on:
   push:
+    branches:
```

Comment:
I want to only execute this GitHub action when releasing a new version, not on every push to the master branch, so this change would have to be reversed

Reply:
Right, I just followed the template from docker; I have no idea what is needed and why it would work; You could decide to push a `:dev` or `:master` is what the intend here is; but sure, we can drop it :)